kornia/feature/responses.py

```diff
@@ -251,6 +263,25 @@ def dog_response(input: Tensor) -> Tensor:
     return input[:, :, 1:] - input[:, :, :-1]
 
 
+def dog_response_single(input: Tensor, sigma1: float = 1.0, sigma2: float = 1.6) -> Tensor:
```

Comment:
Why not jus updating the old one with this new approach to have less to maintain?